### ISSUE TITLE
weird permission problem

### ISSUE THREAD

**Issue by undeflife** [winnfsd/vagrant-winnfsd#38]
Windows 8.1
VirtualBox 4.3.20
Vagrant 1.7.1
vagrant-winnfsd 1.0.11

set winnfsd.uid and winnfsd.gid to 1000 (vagrant)
vim or other operation seems fine but something went wrong when use git

```
$ git add -u
 fatal: error when closing sha1 file: Permission denied

$ git gc
fatal: fsync error on '.git/objects/pack/tmp_pack_1aPwSL': Permission denied
error: failed to run repack
```

.git/objects/pack/tmp_pack\* permission is 555 (r-x)
winnfsd logging
NFS WRITE .git\objects\pack\tmp_pack_1aPwSL Permission denied ACCESS

**Comment by create-renegr** [comment]
same here!

**Comment by grizzlylab** [comment]
Same here :(
Everything related to git give me permission problems...

**Comment by grizzlylab** [comment]
For the record, it is winnfsd related:
https://bitbucket.org/jankowfsky/winnfsd/issue/9/rename-file-on-vm-does-not-work-previously

**Comment by xtreamwayz** [comment]
I had the same issue and nothing worked for me until I tried a test version of winnfsd. I've been using it for a few hours now and it looks like all git related permission denied errors are gone.

The test version is linked in this post: https://bitbucket.org/jankowfsky/winnfsd/issue/9/rename-file-on-vm-does-not-work-previously#comment-17261362

**Comment by rdpascua** [comment]
https://github.com/GM-Alex/vagrant-winnfsd/issues/47

**Comment by tonygriv** [comment]
Same here!

**Comment by petemolinero** [comment]
Me as well :-(

**Comment by luisarc** [comment]
Me too

**Comment by marcharding** [comment]
This should all be fixed with v1.2.0. Please update and try again.

**Comment by loiclavoie** [comment]
This error happen in the current version (1.4) with Windows 10

**Comment by JonathonReinhart** [comment]
> This should all be fixed with v1.2.0. Please update and try again.

@marcharding Any clues as to what the fix was? I'm seeing this same sort of error on something unrelated to NFSd and it would be helpful to understand what the problem was and how you fixed it. Thanks.
